sysconfig/daint/packages.yaml

```diff
@@ -41,12 +41,12 @@ packages:
         version: [system]
     cmake:
         paths:
-            cmake@3.8.1: /apps/common/UES/jenkins/SLES12/easybuild/software/CMake/3.8.1
+            cmake@3.12.4: /apps/daint/UES/jenkins/6.0.UP07/mc/easybuild/software/CMake/3.12.4
         buildable: False
-        version: [3.8.1]
+        version: [3.12.4]
     boost:
-        modules:
```

Comment:
You should be able to have both modules and paths in here now. I made some changes to use both…

Reply:
Ok. But the reason I removed modules is that Cray modules handing with Spack is [bit special](https://spack.readthedocs.io/en/latest/getting_started.html#cray-compiler-detection). The full programming environment is swapped if we load/change modules (like lmod). I was using gcc compiled boost and then everything was changed to GCC in the background.

Reply:
Hm, thanks for the explanation. Seems like we should look out for that when switching to lmod, then.